src/index.ts

```diff
@@ -71,6 +97,18 @@ app.get('/version', (_, response) => {
   return response.send(process.env.VERSION || 'undefined');
 });
 
+app.post('/shout', urlencodedParser, (request, response) => {
```

Comment:
Given that we're not planning to use the `shout` in the API response to trigger the vulnerability anymore, I wonder if we might want to modify the naming convention here so as not to confuse the two?

Reply:
if it's not too confusing I think it might be good to have both paths work to trigger the same behavior

Reply:
@samueljohnwilson Renamed in e3b6fc1